scripts/exgfs_wave_nawips.sh

```diff
@@ -102,7 +102,7 @@ while [ $fhcnt -le $FHMAX_WAV ]; do
                                           $GRIBIN 1> out 2>&1
       OK=$?
       if [ "$OK" != '0' ]; then 
-        echo "ABNORMAL EXIT: ERROR IN interpolation the global grid"
+        msg="ABNORMAL EXIT: ERROR IN interpolation the global grid"
```

Comment:
Where does `msg` go after this?

Reply:
it gets echo'd at the end, then I think it is used by `err_chk`. Problem was that `msg` wasn't defined, so it would exit with the unbound variable.

Reply:
Got it, thanks!